rpc/wrtc_server_stream.go

```diff
@@ -50,12 +54,21 @@ func newWebRTCServerStream(
 	onDone func(id uint64),
 	logger utils.ZapCompatibleLogger,
 ) *webrtcServerStream {
+	requestID := uuid.New().String()
+
+	ctx = context.WithValue(ctx, RequestID{}, requestID)
```

Comment:
let's punt adding request ids to contexts to a later change. we shouldn't be adding them without checking if a request id already exists on the request (in many cases a client should be generating the request id), and right now the way we're adding them seem a bit haphazard